src/ansible/roles/keycloak/tasks/main.yml

```diff
@@ -0,0 +1,148 @@
+- name: print keycloak_url
```

Comment:
Uppercase P

Reply:
I think I changed all the task names to start with uppercase.